packages/api/lib/writeRecords/write-granules.js

```diff
@@ -835,13 +842,29 @@ const writeGranulesFromMessage = async ({
   const executionDescription = await granuleModel.describeGranuleExecution(executionArn);
   const processingTimeInfo = getExecutionProcessingTimeInfo(executionDescription);
   const provider = getMessageProvider(cumulusMessage);
-  const workflowStartTime = getMessageWorkflowStartTime(cumulusMessage);
   const error = parseException(cumulusMessage.exception);
   const workflowStatus = getMetaStatus(cumulusMessage);
   const collectionId = getCollectionIdFromMessage(cumulusMessage);
   const pdrName = getMessagePdrName(cumulusMessage);
   const queryFields = getGranuleQueryFields(cumulusMessage);
 
+  let workflowStartTime;
+  try {
```

Comment:
This is defensively adding a catch explicitly to this function instead of relying solely on getMessageWorkflowStartTime throwing, in addition to more explicit logging.